bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -26,11 +26,13 @@ f = h5py.File(args.output_file, "w")
 logging.info('Copying segments and attributes to %s' % args.output_file)
 # Move segments information into the final file - remove some duplication in earlier files
 for fi in files:
-    for key in f['segments']:
-        f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
-        f['segments/%s/start' % key] = fi['segments/%s/start' % key][:]
+    for key in fi['segments']:
         if key.startswith('foreground') or key.startswith('background'):
             continue
+        f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
+        f['segments/%s/start' % key] = fi['segments/%s/start' % key][:]
+        f['segments/%s/foreground_veto/end' % key] = fi['segments/foreground_veto/end'][:]
```

Comment:
the `fi['segments/foreground_veto/end']` and `fi['segments/foreground_veto/start']` needs to be in a test for foreground_veto, as this will be present in full_data, but not in injections

Reply:
done